quickstarts/taskqueues-backup-images/functions/index.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for t`he specific language governing permissions and
+ * limitations under the License.
+ */
+// [START import]
+"use strict";
+const path = require("path");
+const fetch = require("node-fetch");
+const functions = require("firebase-functions");
+const {initializeApp} = require("firebase-admin/app");
+const {getFunctions} = require("firebase-admin/functions");
+const {getStorage} = require("firebase-admin/storage");
+const logger = functions.logger;
+const HttpsError = functions.https.HttpsError;
+initializeApp();
+// [END import]
+
+
+// [START backupImages]
+/**
+ * Grabs Astronomy Photo of the Day (APOD) using NASA's API.
+ *
+ */
+exports.backupApod = functions
```

Comment:
v2 only supports lowercase function names.

Reply:
technically, this is v1 task queue functions. Should I be just targeting the main branch in that case?

Reply:
Oh, my bad, I didn't know this was available in v1. I'll wait for Jeff to confirm which branch to target.